src/main/java/me/totalfreedom/totalfreedommod/command/Command_manageshop.java

```diff
@@ -1,209 +1,207 @@
-                        shopData.setCoins(amount);
-                        plugin.sh.save(shopData);
-                        msg("Successfully set " + player.getName() + "'s coins to " + amount, ChatColor.GREEN);
-                        player.sendMessage(ChatColor.GREEN + sender.getName() + " set your coin balance to " + amount);
-                        return true;
-                    }
-                    catch (NumberFormatException ex)
-                    {
-                        msg("Invalid number: " + args[2], ChatColor.RED);
-                        return true;
-                    }
-            }
-        }
-        else if (args[0].equals("items"))
-        {
-            if (args[1].equals("list"))
-            {
-                msg("List of all shop items: " + StringUtils.join(ShopItem.values(), ", "));
-                return true;
-            }
-
-            if (args.length < 4)
-            {
-                return false;
-            }
-
-            if (args[1].equals("give"))
-            {
-                ShopItem item = ShopItem.findItem(args[2].toUpperCase());
-                if (item == null)
-                {
-                    msg(args[2] + " is not a valid item.", ChatColor.RED);
-                    return true;
-                }
-
-                Player player = getPlayer(args[3]);
-                if (player == null)
-                {
-                    msg(PLAYER_NOT_FOUND);
-                    return true;
-                }
-
-                ShopData shopData = plugin.sh.getData(player);
-                shopData.giveItem(item);
-                plugin.sh.save(shopData);
-                msg("Successfully gave the " + item.getName() + " to " + player.getName(), ChatColor.GREEN);
-                player.sendMessage(ChatColor.GREEN + sender.getName() + " gave the " + item.getName() + " to you");
-                return true;
-            }
-            else if (args[1].equals("take"))
-            {
-                ShopItem item = ShopItem.findItem(args[2].toUpperCase());
-                if (item == null)
-                {
-                    msg(args[2] + " is not a valid item.", ChatColor.RED);
-                    return true;
-                }
-
-                Player player = getPlayer(args[3]);
-                if (player == null)
-                {
-                    msg(PLAYER_NOT_FOUND);
-                    return true;
-                }
-
-                ShopData shopData = plugin.sh.getData(player);
-                shopData.removeItem(item);
-                plugin.sh.save(shopData);
-                msg("Successfully took the " + item.getName() + " from " + player.getName(), ChatColor.GREEN);
-                player.sendMessage(ChatColor.RED + sender.getName() + " took the " + item.getName() + " from you");
-                return true;
-            }
-
-        }
-        return false;
-    }
-
-}
+package me.totalfreedom.totalfreedommod.command;
```

Comment:
This entire file hasn't changed, and doing this is gonna break the Git history (OR make it difficult to work with)